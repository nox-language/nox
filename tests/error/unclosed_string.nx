fun main() = (
    var N = 8;
    print(")
)
