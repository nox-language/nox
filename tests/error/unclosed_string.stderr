error: Unclosed string
   --> tests/error/unclosed_string.nx:3:11
  |
3 |     print(")
  |           ^

